server/controllers/ChallengeController.js

```diff
@@ -1,10 +1,37 @@
 const db = require("../models/index");
 const Challenge = db.Challenge;
 
-exports.get = async (req, res) => {
+exports.findOne = async (req, res) => {
```

Comment:
Why `findOne` instead of  `get`?

Reply:
We may want to fetch many challenges at once. Get is ambiguous as to how many challenges we are fetching.